src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  height: 100vh;
+  padding: $padding;
+  margin: 0 auto;
+  width: $calendar-width;
+  list-style: none;
+
+  display: flex;
+  align-items: center;
+  align-content: center;
+  gap: $gap-between-boxes;
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: #eee;
+    width: $day-box-size;
+    height: $day-box-size;
+    border: 1px solid;
+
+    cursor: pointer;
+    transition: all 0.5s;
```

Comment:
It's better to specify to what properties you apply the transition.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/4c66f134-2981-4dd9-ab6d-46fc03980cfc)